flow/envs/base_env.py

```diff
@@ -349,8 +314,20 @@ def setup_initial_state(self):
         self.vehicles.update(vehicle_obs, id_lists, self)
         self.traffic_lights.update(tls_obs)
 
-        # store the network observations in the vehicles class
-        self.vehicles.update(vehicle_obs, id_lists, self)
+        # check to make sure all vehicles have been spawned
+        if len(self.initial_ids) < self.vehicles.num_vehicles:
+            logging.error("Not enough vehicles have spawned! Bad start?")
+            exit()
```

Comment:
Shouldn't this use `sys.exit` and return a nonzero value?

Reply:
This is correct, thanks for the catch!

Reply:
@eugenevinitsky 

No problem!